docs/source/community/pull_requests.rst

```diff
@@ -0,0 +1,360 @@
+.. Copyright 2022 NWChemEx-Project
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+.. http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+
+.. _prs_and_nwx:
+
+###############################################
+NWChemEx Community Guidelines for Pull Requests
+###############################################
+
+TL;DR PRs are the most natural way to update a project hosted on GitHub. We
+have created a PR template to help PR authors include enough detail for
+reviewers to do their job. And we have suggested workflows for authors and
+reviewers.
+
+*************
+What is a PR?
+*************
+
+A PR is a GitHub feature used to contribute changes to a GitHub repo. More
+specifically, each GitHub repo contains one or more branches. When you
+want to modify one of those branches you:
+
+#. Fork the target repo on GitHub
```

Comment:
I think we had some simple instructions that covered some of this in documentation - although I can't find it at the moment.  If so, it would be good to point to that for these instructions.

Reply:
removed the explicit steps and added a link.